ui/core/proto_utils/action_id.ts

```diff
@@ -378,6 +378,16 @@ export class ActionId {
 				break;
 			case 'Fulmination':
 				name += ` (${this.tag + 3})`
+			case 'Moonfire':
+				if (this.tag == 1) {
+					name += ' (DoT)'
+				}
+				break;
```

Comment:
you can have the switch case falltrough to the next case so just putting:
```
case 'Moonfire':
case 'Sunfire':
  dot...
```

Reply:
oh true, thanks